api/src/main/java/org/openmrs/module/idgen/service/BaseIdentifierSourceService.java

```diff
@@ -397,7 +397,7 @@ public void setProcessors(Map<Class<? extends IdentifierSource>, IdentifierSourc
     public void checkAndRefillIdentifierPool(IdentifierPool pool){
         if (pool.getSource() != null && (pool.getSource() instanceof SequentialIdentifierGenerator || pool.getSource() instanceof RemoteIdentifierSource)) {
             while (pool.getMinPoolSize() > getQuantityInPool(pool, true, false)){
-                addIdentifiersToPool(pool, Integer.valueOf(pool.getBatchSize()));
```

Comment:
You also seem to have forgotten what was reverted in IdentifierSource.hbm.xml

Reply:
Let me counter check